src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -436,6 +485,24 @@ internal void InvokeResponse(InvocationResponse invokeResponse)
             }
         }
 
+        internal void CloseSharedMemoryResourcesForInvocation(string invocationId, IList<string> outputMaps)
```

Comment:
This method is doing two things:
1) Freeing shared memory resources (if any) that were created by the host.
2) Sending a message to the worker to close (if any) maps that were created by the worker.

Note that only (2) involves any *Send*ing so I would say it would be a bit misleading to rename this method to *Send*.